djstripe/utils.py

```diff
@@ -108,3 +108,17 @@ def get_model(model_name):
 def get_queryset(pks, model_name):
     model = get_model(model_name)
     return model.objects.filter(pk__in=pks)
+
+
+def get_timezone_utc():
+    """
+    Returns UTC attribute in a backwards compatible way.
+
+    UTC attribute has been moved from django.utils.timezone module to
+    datetime.timezone class
+    """
+    try:
+        return timezone.utc
+    except AttributeError:
+        # Django 4+
+        return datetime.timezone.utc
```

Comment:
Flip the order. You should return datetime.timezone.utc by default, if attributeerror return django's.